specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2020-09-01-preview/inference.json

```diff
@@ -0,0 +1,1348 @@
+          }
+        }
+      }
+    },
+    "InferenceSku": {
+      "description": "Sku of the resource",
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "Name of the sku",
+          "type": "string",
+          "enum": [
+            "Standard_D3_v2",
+            "Standard_D2_v2"
+          ],
+          "x-ms-enum": {
+            "name": "allowableSkus",
+            "modelAsString": true
+          }
+        },
+        "tier": {
+          "type": "string",
+          "enum": [
+            "Basic"
+          ],
+          "x-ms-enum": {
+            "name": "allowedSkuTiers",
+            "modelAsString": true
+          },
+          "description": "The tier of the sku."
+        }
+      }
+    },
+    "SkuDiscoveryInfo": {
+      "description": "SKU discovery information.",
+      "type": "object",
+      "properties": {
+        "resourceType": {
+          "description": "Resource type that this SKU applies to.",
+          "type": "string"
+        },
+        "sku": {
+          "$ref": "#/definitions/InferenceSku",
+          "description": "Name and tier of the SKU."
+        },
+        "capacity": {
+          "$ref": "#/definitions/SkuScaleSettings",
+          "description": "Min, max, and default scale values of the SKU."
+        }
+      }
+    },
+    "SkuDiscoveryInfoCollection": {
+      "description": "Collection of SKU information.",
+      "required": [
+        "value"
+      ],
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Collection of resources.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SkuDiscoveryInfo"
+          }
+        }
+      }
+    },
+    "InferenceEndpointDeploymentSku": {
+      "description": "Description of a SKU for a scalable resource.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/InferenceSku"
+        },
+        {
+          "type": "object",
+          "properties": {
+            "capacity": {
+              "description": "Current number of instances assigned to the resource.",
+              "type": "integer",
+              "format": "int32"
+            }
+          }
+        }
+      ]
+    },
+    "InferenceEndpointsUpdateParameters": {
+      "description": "The parameters for updating an inference endpoint.",
+      "properties": {
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "Resource tags."
+        }
+      }
+    },
+    "InferenceEndpointDeploymentUpdateParameters": {
+      "description": "The parameters for updating an inference deployment.",
+      "properties": {
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "Resource tags."
+        },
+        "sku": {
```

Comment:
cannot update SKU

Reply:
need this for azMon autoscale to change capacity.